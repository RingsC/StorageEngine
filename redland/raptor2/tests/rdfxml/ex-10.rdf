<?xml version="1.0" encoding="utf-8"?>
<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
         xmlns="http://purl.org/rss/1.0/">
   <!-- Check about rdf:li and rdf:resource -->
   <channel rdf:about="http://example.org/channel/">
      <items>
         <rdf:Seq>
            <rdf:li rdf:resource="http://example.org/01"/>
            <rdf:li rdf:resource="http://example.org/02"/>
            <rdf:li rdf:resource="http://example.org/03"/>
         </rdf:Seq>
      </items>
   </channel>
</rdf:RDF>
