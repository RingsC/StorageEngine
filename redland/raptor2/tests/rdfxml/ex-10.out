<http://example.org/channel/> <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://purl.org/rss/1.0/channel> .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#type> <http://www.w3.org/1999/02/22-rdf-syntax-ns#Seq> .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#_1> <http://example.org/01> .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#_2> <http://example.org/02> .
_:genid1 <http://www.w3.org/1999/02/22-rdf-syntax-ns#_3> <http://example.org/03> .
<http://example.org/channel/> <http://purl.org/rss/1.0/items> _:genid1 .
